.github/workflows/run_release.yaml

```diff
@@ -36,20 +36,109 @@ jobs:
     name: Check whether to release
     if: |
       github.event_name == 'workflow_dispatch' ||
-      github.event_name == 'release' ||
       (
         github.event_name == 'push' &&
         !startsWith(github.event.head_commit.message, 'docs') &&
```

Comment:
```suggestion
        !startsWith(github.event.head_commit.message, 'doc') &&
```

Reply:
the commit prefix is called `docs:` not `doc:`

https://www.conventionalcommits.org/en/v1.0.0/#summary

Reply:
Oh, I based this on what we have in `cliff.toml`, in that case, it should be updated there from `doc` to `docs`.

Reply:
I suspect that git-cliff only puts `^doc` in the default configuration to catch both `doc:` and `docs:` so that they please everyone :smile: 

The newly added PR title check should prevent any errors anyway, but I can update `cliff.toml` too.